eng/illink.targets

```diff
@@ -57,19 +60,31 @@
   -->
   <ItemGroup Condition="'$(BinPlaceILLinkTrimAssembly)' == 'true'">
     <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/trimmed</RuntimePath>
+      <RuntimePath>$(ILLinkTrimAssemblyArtifactsRootDir)/trimmed</RuntimePath>
       <ItemName>TrimmedItem</ItemName>
     </BinPlaceTargetFramework>
     <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/reports</RuntimePath>
+      <RuntimePath>$(ILLinkTrimAssemblyArtifactsRootDir)/reports</RuntimePath>
       <ItemName>TrimmingReport</ItemName>
     </BinPlaceTargetFramework>
     <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/pretrimmed</RuntimePath>
+      <RuntimePath>$(ILLinkTrimAssemblyArtifactsRootDir)/pretrimmed</RuntimePath>
       <ItemName>PreTrimmedItem</ItemName>
     </BinPlaceTargetFramework>
   </ItemGroup>
 
+  <ItemGroup>
+    <BinPlaceTargetFrameworks Include="$(NetCoreAppCurrent)">
+      <RuntimePath>$(ILLinkTrimAssemblySuppressionsXmlsDir)</RuntimePath>
+      <ItemName>ILLinkSuppressionsXmls</ItemName>
+    </BinPlaceTargetFrameworks>
+  </ItemGroup>
+
+  <ItemGroup>
+    <ILLinkSuppressionsXmls Condition="Exists('$(ILLinkSuppressionsXmlFile).xml')" Include="$(ILLinkSuppressionsXmlFile).xml" />
+    <ILLinkSuppressionsXmls Update="@(ILLinkSuppressionsXmls)" TargetPath="%(FileName).$(AssemblyName).xml" />
```

Comment:
Can't you set the `TargetPath` when doing the Include?

```xml
<ILLinkSuppressionsXmls Include="$(ILLinkSuppressionsXmlFile).xml"
                        Condition="Exists('$(ILLinkSuppressionsXmlFile).xml')"
                        TargetPath="%(FileName).$(AssemblyName).xml"  />
```

Reply:
Now that there are multiple files included above, setting the `TargetPath` once with this line seems a bit cleaner than setting it for each file.